dobi.yaml

```diff
@@ -6,3 +6,73 @@ image=image-kas:
   image: ghcr.io/siemens/kas/kas
   tags: ['2.3.3']
   pull: once
+
+image=image-curl-jq:
+  image: dwdraju/alpine-curl-jq
+  tags: [latest]
+  pull: once
+
+image=image-mender-cli:
+  image: ci4rail/mender-cli
```

Comment:
Why using own meder cli image?

Reply:
Because there is none from mender on docker hub. And 2nd: The Dockerfile provided by mender did not work.
Already created a PR at mender